pkg/iptables/iptables.go

```diff
@@ -188,3 +196,29 @@ func UpdateChainRules(ipt Interface, table, chain string, rules [][]string) erro
 
 	return nil
 }
+
+// iptables mode should be retrieved from iptables library (when supported )
+
+func IsNftMode(ipt Interface) bool {
```

Comment:
This function doesn't use `ipt`.  It seems this function should be part of the `Interface`?

Reply:
Yes you are right, I thought to include it in the interface to make it easier to retrieve mode from iptables library when supported.

Both options seem fine to me, would you rather exclude it from the interface?

Reply:
I'd prefer on the interface so we can mock it in unit tests.